CExts.c

```diff
@@ -0,0 +1,59 @@
+//
+//  CExts.cpp
+//  XcodeGen
+//
+//  Created by Michael Eisel on 3/11/20.
+//
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <ctype.h>
+#include <string.h>
+#include <stdbool.h>
+
+const char *PATEscapedString(const char *string) {
```

Comment:
Would you mind documenting this method? What does `PAT` stand for?

Reply:
PAT is an arbitrary 3-letter prefix for namespacing. What would you prefer, XCP?